scripts/generate-docs.sh

```diff
@@ -5,13 +5,13 @@
 
 exclude_files=()
 
-# Check if manual changes were made to any excluded files and exit
-# otherwise these will be lost with `tfplugindocs`
-if [ "$(git status --porcelain "${exclude_files[@]}")" ]; then
-  echo "Uncommitted changes were detected to the following files. These aren't autogenerated, please commit or stash these changes and try again"
-  echo $(git status --porcelain "${exclude_files[@]}")
-  exit 1
-fi
+## Check if manual changes were made to any excluded files and exit
```

Comment:
Do you really need to keep this commented out here?

Reply:
nice thnks